arm-sql/2015-05-01-preview/swagger/automaticTuning.json

```diff
@@ -258,7 +227,10 @@
             "Inherit",
             "Custom"
           ],
-          "type": "string"
+          "type": "string",
+          "x-ms-enum": {
+            "name": "AutomaticTuningMode"
```

Comment:
This is another issue that has never been brought up with us before.  I'll modify the generator since now we want explicit "modelAsString": "false"